web/modules/custom/bnf/tests/src/Unit/Mapper/ParagraphAccordionMapperTest.php

```diff
@@ -30,33 +30,32 @@ protected function getEntityClass(): string {
   }
 
   /**
-   * Test text paragraph mapping.
+   * Test paragraph mapping.
    */
-  public function testParagraphTextBodyMapping(): void {
+  public function testParagraphAccordionMapping(): void {
     $this->storageProphecy->create([
       'type' => 'accordion',
+      'field_accordion_title' => [
+        'value' => 'This is the title',
+        'format' => 'format1',
+      ],
+      'field_accordion_description' => [
+        'value' => 'This is the description',
+        'format' => 'format2',
+      ],
     ])->willReturn($this->entityProphecy);
 
     $mapper = new ParagraphAccordionMapper([], '', [], $this->entityManagerProphecy->reveal());
 
-    $graphqlArticle = ParagraphAccordion::make(
+    $graphqlElement = ParagraphAccordion::make(
+      'accordion',
       TitleText::make('This is the title', 'format1'),
       DescriptionText::make('This is the description', 'format2'),
     );
 
-    $paragraph = $mapper->map($graphqlArticle);
+    $paragraph = $mapper->map($graphqlElement);
 
     $this->assertSame($paragraph, $this->entityProphecy->reveal());
-
-    $this->entityProphecy->set('field_accordion_title', [
```

Comment:
This assertion has been removed, as I place the code directly in the  `create` method, rather than dedicated `set`'s in the mapper.

Reply:
And why do you do that? Using `set` have the advantage that you can separately test for each field, and when a test fails it'll tell you which field wasn't set properly, not that the whole create failed.

Reply:
Good point, I'll move it back in the accordion and textbody mappers, as it was before. I think the reason i prefered doing it all in `create` was that to make the mappers more readable.
I think you're right, that we should use `set` when we also have unit tests.